src/types.h

```diff
@@ -53,13 +53,20 @@ class Input : public ArgList {
 template<typename T>
 struct Converter {
     /** Parse a text into the right type
-     * Throws std::invalid_argument or std::out_of_range
-     * 'Source' may be given relative to 'relativeTo', e.g. "toggle" for booleans.
+     * @throws std::invalid_argument or std::out_of_range
+     * @param source verbal description of a value
      */
-    static T parse(const std::string& source, T const* relativeTo);
-    static T parseDefault(const std::string& source, const T& relativeTo) {
-        return parse(source, &relativeTo);
-    }
+    static T parse(const std::string& source);
+    /** Parse a text into the right type
+     * @brief overload that accepts a reference value to relate input string to
+     * @param source may be given relative to 'relativeTo', e.g. "toggle" for booleans.
+     * @param relativeTo previous/existing value
+     * @throws std::invalid_argument or std::out_of_range
+     */
+    static T parse(const std::string& source, const T& relativeTo) {
+        // note: specializations _do_ use relativeTo where appropriate
+        return parse(source);
+    };
```

Comment:
We only use the `realtiveTo` for booleans (so far), so this is a very sensible compromise! :-)